packages/vite/src/node/__tests__/fixtures/scan-jsx-runtime/entry-jsx.tsx

```diff
@@ -0,0 +1,5 @@
+;(globalThis as any).__test_scan_jsx_runtime ??= 0
+;(globalThis as any).__test_scan_jsx_runtime++
+
+// @ts-ignore
+export default <div>test</div>
```

Comment:
[nit] Maybe using `preact` can remove this `@ts-ignore`?

Reply:
It looks like Vue doesn't support `children`. I changed it to `<div />` and no red squiggle.